libcxx/include/__tree

```diff
@@ -2033,7 +2059,8 @@ __tree<_Tp, _Compare, _Allocator>::__erase_multi(const _Key& __k) {
 
 template <class _Tp, class _Compare, class _Allocator>
 template <class _Key>
-typename __tree<_Tp, _Compare, _Allocator>::iterator __tree<_Tp, _Compare, _Allocator>::find(const _Key& __v) {
+typename __tree<_Tp, _Compare, _Allocator>::iterator
+__tree<_Tp, _Compare, _Allocator>::find(__multi_tag, const _Key& __v) {
```

Comment:
Can we inline these functions into the `__tree` definition? That'll make `find(__multi` and `find(__unique` more similar.